dbms/include/DB/Common/UInt128.h

```diff
@@ -8,6 +8,11 @@
 namespace DB
 {
 
+#if !__clang__
```

Comment:
Aggregator.cpp already have it -
https://github.com/yandex/ClickHouse/blob/master/dbms/src/Interpreters/Aggregator.cpp#L574